Networking/Networking/Model/Product/Product.swift

```diff
@@ -117,7 +117,7 @@ public struct Product: Codable, GeneratedCopiable, Equatable, GeneratedFakeable
     // MARK: Subscription Product properties
 
     /// Subscription settings. Applicable to subscription-type products only.
-    public let subscription: ProductSubscription?
+    public let subscription: ProductSubscription? //TODO: migrate AddOns to MetaData
```

Comment:
np: should say Subscriptions instead of AddOns?

Reply:
Yes, good catch. Edited here 61bd5ab44e72071fd0d0545831e2c0f7121251db